components/count-badge/count-badge-mixin.js

```diff
@@ -183,9 +188,20 @@ export const CountBadgeMixin = superclass => class extends LocalizeCoreElement(S
 			numberString = formatNumber(numberString);
 		}
 
+		return numberString;
+	}
+
+	renderCount(numberStyles) {
```

Comment:
Extracted this logic into a public function so the icon badge can determine the length of the number string.